dist/colors.scss

```diff
@@ -488,109 +488,101 @@ $blk-235: #060606;
 $blk-240: #000000;
 
 // UI Colors Light Theme
-$ui-blue-1: $h-bb-060;
-$ui-blue-2: $h-bb-050;
-$ui-blue-3: $h-bb-070;
-$ui-green-1: $h-gg-060;
-$ui-green-2: $h-gg-050;
-$ui-green-3: $h-gg-070;
-$ui-yellow-1: $h-yy-060;
-$ui-yellow-2: $h-yy-050;
-$ui-yellow-3: $h-yy-070;
-$ui-red-1: $h-rr-060;
-$ui-red-2: $h-rr-050;
-$ui-red-3: $h-rr-070;
+$ui-brand: $h-bb-060;
+$ui-brand-hover: $h-bb-070;
+$ui-brand-press: $h-bb-080;
 $ui-background: $blk-005;
 $ui-foreground-1: $blk-000;
 $ui-foreground-2: $blk-010;
 $ui-foreground-3: $blk-020;
 $ui-text-1: $blk-220;
 $ui-text-2: $blk-170;
 $ui-text-3: $blk-140;
+$ui-text-inverse: $blk-000;
+$ui-text-link: $h-bb-070;
 $ui-border-1: $blk-050;
 $ui-border-2: $blk-030;
 $ui-border-3: $blk-020;
-$ui-border-4: $blk-090;
-$ui-border-5: $blk-130;
+$ui-border-input: $blk-100;
+$ui-info: $h-bb-070;
+$ui-success: $h-gg-060;
+$ui-warning: $h-yy-060;
+$ui-danger: $h-rr-060;
+$ui-danger-hover: $h-rr-070;
+$ui-danger-press: $h-rr-080;
```

Comment:
Per discussion in Teams: remove intermediate vars and assign palette variables directly to CSS vars.